src/dune_rules/compilation_context.ml

```diff
@@ -130,7 +130,7 @@ let create
   ?stdlib
   ~js_of_ocaml
   ~package
-  ?public_lib_name
+  ~melange_package_name
```

Comment:
What was behind the decision of changing from optional param to non-optional?

Reply:
To make it explicit where we’re passing the argument